### PR TITLE
Updated Vagrant file as it wasn't working.

### PR DESCRIPTION
**Pull request by ERYoung11**
## Description

I updated the vagrant file to create .rpm and .deb files.  It works for Python 3.

## Checklist

- [ ] I've added this contribution to the `changelog.rst`.
- [ ] I've added my name to the `AUTHORS` file (or it's already there).

- [ X] I installed pre-commit hooks (`pip install pre-commit && pre-commit install`), and ran `black` on my code.
- [x] Please squash merge this pull request (uncheck if you'd like us to merge as multiple commits)

### CODE REVIEW

**Comment by ERYoung11** [comment]
My change didn't cause these timeout failures.

File: setup.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 install_requirements = [
     "pgspecial>=1.11.8",
     "click >= 4.1",
+    "packaging>=20.9",
```
Comment (j-bennet) [inline setup.py:11]: Why do we need this one?

Comment (ERYoung11) [reply setup.py:11]: Without it, we see the following error when we try to run pgcli after installation.

_Traceback (most recent call last):
  File "/usr/local/bin/pgcli", line 5, in <module>
    from pgcli.main import cli
  File "/usr/share/pgcli/lib/python3.7/site-packages/pgcli/main.py", line 56, in <module>
    from .pgtoolbar import create_toolbar_tokens_func
  File "/usr/share/pgcli/lib/python3.7/site-packages/pgcli/pgtoolbar.py", line 1, in <module>
    from packaging.version import parse as parse_version
ModuleNotFoundError: No module named 'packaging'_

Comment (ERYoung11) [reply setup.py:11]: It looks like PR #1246 added in the packaging module.

Comment (pasenor) [reply setup.py:11]: It was a mistake that is addressed in #1252. Packaging is a build dependency, but should not be needed at run time.

Comment (ERYoung11) [reply setup.py:11]: > It was a mistake that is addressed in #1252. Packaging is a build dependency, but should not be needed at run time.

I'm certainly not qualified to question you but maybe you can advise how I can do the build differently.  I am building from the latest master branch.  When it builds, I then install the .rpm or .deb on a system.

If I don't include packaging in setup.py, it complains that it doesn't have it when I try to run it.  

Should I do something different?

Comment (ERYoung11) [reply setup.py:11]: oh duh, I see that your pull request is still open :)

Ok, once you merge that I'll update this pull request.

**Comment by ERYoung11** [comment]
OK @pasenor & @j-bennet  I have now removed the packaging entry from setup.py as we discussed above.   I reran the builds and they worked for both centos and debian.  I installed the resulting .deb and .rpm and both of those worked.

I think this is ready for merging now.

**Comment by j-bennet** [comment]
@ERYoung11 Was this closed accidentally?

**Comment by ERYoung11** [comment]
> @ERYoung11 Was this closed accidentally?

So, here's where my ignorance starts to show.  It looked like you had already merged my change so I was just trying to clean up.  I must have misunderstood what was going on.  I though I saw the change reflected in the master branch so I thought you were done.

Go ahead and do whatever is needed to merge the change.

**Comment by j-bennet** [comment]
@ERYoung11 Github is having issues today. I pressed the "merge" button, but got an error response. I do see your commits in master, so the merge must have gone through. 🤷  Closing again!

**Comment by ERYoung11** [comment]
> @ERYoung11 Github is having issues today. I pressed the "merge" button, but got an error response. I do see your commits in master, so the merge must have gone through. 🤷 Closing again!

Good to know because I checked like 3 times to be sure it was merged but then it said it had unmerged commits so I was pretty confused.

**Pull request closed without merging**